src/main/resources/META-INF/plugin.xml

```diff
@@ -100,4 +100,10 @@
               description="Snyk: Show Low severity issues"/>
     </group>
   </actions>
+
+  <extensionPoints>
+    <extensionPoint
+      name="controllerLifecycleManager"
```

Comment:
how about `SnykControllerManager`? I'm not sure that in all searches and portals, the plugin the extension point comes from is highly visible.